src/test/java/com/synacy/lesson01/exercises/InvoiceTest.java

```diff
@@ -0,0 +1,29 @@
+package com.synacy.lesson01.exercises;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+import java.util.HashSet;
+import java.util.Set;
+
+import static org.junit.Assert.*;
+
+
+public class InvoiceTest {
+
+    @Test
+    public void getTotalAmount_shouldReturnTotalAmountOfAllItemsInTheInvoice() throws Exception {
+        Product product = new Product("sample product", 200.00);
+        InvoiceItem invoiceItem = new InvoiceItem(product, 2);
+
+        Set<InvoiceItem> items = new HashSet<>();
+        items.add(invoiceItem);
+
```

Comment:
When dealing with methods manipulating `Collection`s, it would be appropriate if we try things out with multiple elements. For this particular instance, having only one `InvoiceItem` in our `Invoice.items` does not really provide us with confidence that the extended prices are correctly summed up since we only have one item; having at least two items would have provided us with the idea that the summation was done well among the element in the collection.

This is generally recommended when testing methods that manipulate collections - either aggregation (sum, averages), mappings (squaring each element in the collection), and filtering (looking for elements that meet certain criteria).